tests/scripts/thread-cert/pktverify/layer_fields.py

```diff
@@ -530,8 +530,7 @@ def __call__(self, v: Union[LayerFieldsContainer, LayerField]):
     'thread_nwd.tlv.server.16': _auto,
     'thread_nwd.tlv.border_router.16': _auto,
     'thread_nwd.tlv.sub_tlvs': _list(_str),
-    'thread_nwd.tlv.prefix.length': _auto,
-    'thread_nwd.tlv.prefix.domain_id': _auto,
+    'thread_nwd.tlv.prefix': _list(_str),
```

Comment:
Better to parse prefix as `Ipv6Addr` (i.e. using `_ipv6_addr`) ? 
We don't have `Ip6Prefix` class yet in `pktverify`. I am fine with using `Ip6Addr` for prefixes, but defining `Ip6Prefix` would be even better. 
@FiveDimensions